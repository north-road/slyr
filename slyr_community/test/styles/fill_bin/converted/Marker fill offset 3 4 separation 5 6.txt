[{'class': 'QgsPointPatternFillSymbolLayer',
  'distance_x': '5',
  'distance_x_unit': 'Point',
  'distance_y': '6',
  'distance_y_unit': 'Point',
  'offset_x': '3',
  'offset_x_unit': 'Point',
  'offset_y': '4',
  'offset_y_unit': 'Point',
  'seed': '0',
  'subsymbol': [{'class': 'QgsSimpleMarkerSymbolLayer',
                 'color': '230,0,0,255',
                 'offset_unit': 'Point',
                 'outline_style': 'no',
                 'size': '8',
                 'size_unit': 'Point'}]},
 {'capstyle': 'round',
  'class': 'QgsSimpleLineSymbolLayer',
  'joinstyle': 'round',
  'line_color': '110,110,110,255',
  'line_width': '1',
  'line_width_unit': 'Point'}]
