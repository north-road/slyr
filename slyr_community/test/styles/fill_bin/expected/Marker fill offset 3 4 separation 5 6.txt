{'levels': [{'enabled': True,
             'grid_angle': 0.0,
             'locked': False,
             'marker': {'angle': 0.0,
                        'color': {'B': 0,
                                  'G': 0,
                                  'R': 230,
                                  'dither': False,
                                  'is_null': False,
                                  'stream_offset': '0xa5',
                                  'type': 'RgbColor',
                                  'version': 1},
                        'enabled': True,
                        'locked': False,
                        'marker_type': 'circle',
                        'outline_color': {'B': 0,
                                          'G': 0,
                                          'R': 0,
                                          'dither': False,
                                          'is_null': False,
                                          'stream_offset': '0x109',
                                          'type': 'RgbColor',
                                          'version': 1},
                        'outline_enabled': False,
                        'outline_size': 1.0,
                        'rotate_with_transform': True,
                        'size': 8.0,
                        'stream_offset': '0x93',
                        'symbol_level': 0,
                        'type': 'SimpleMarkerSymbol',
                        'version': 2,
                        'x_offset': 0.0,
                        'y_offset': 0.0},
             'offset_x': 3.0,
             'offset_y': 4.0,
             'outline': {'color': {'B': 110,
                                   'G': 110,
                                   'R': 110,
                                   'dither': True,
                                   'is_null': False,
                                   'stream_offset': '0x14c',
                                   'type': 'RgbColor',
                                   'version': 1},
                         'enabled': True,
                         'line_type': 'solid',
                         'locked': False,
                         'stream_offset': '0x13a',
                         'symbol_level': 0,
                         'type': 'SimpleLineSymbol',
                         'version': 1,
                         'width': 1.0},
             'random': False,
             'separation_x': 5.0,
             'separation_y': 6.0,
             'stream_offset': '0x4d',
             'symbol_level': 0,
             'type': 'MarkerFillSymbol',
             'version': 1}],
 'stream_offset': '0x0',
 'symbol_level': 0,
 'type': 'MultiLayerFillSymbol',
 'version': 2}
